en/releases/README.md

```diff
@@ -1,8 +1,115 @@
+
+## Release Versioning
+
+PX4 uses a Semantic versioning (`Major.minor.patch[-beta#]`) system ([Wikipedia article](https://en.wikipedia.org/wiki/Software_versioning)).
+
+- Change in `Major`: Breaking changes
+- Change in `minor`: New non-breaking changes
+- Change in `patch`: All other non-breaking changes
+
+## Release Schedule
+
+PX4 follows the following release schedule:
+
+<!-- TO BE DEFINED -->
+
+- `Major` releases: By consensus among Maintainers & Community
+- `minor` releases: Roughly every 6 months
+- `patch` releases: When critical bug-fix / hardware support is needed
+- `beta` releases: Every 1 week during beta testing phase
+
+### Past Major/Minor releases
+
+There has been roughly 14 minor release in the past, with roughly 6 months ~ 1 year between new minor version releases. And there hasn't been any new major version releases yet.
+
+- [v1.13.0](https://github.com/PX4/PX4-Autopilot/releases/tag/v1.13.0): June 22nd, 2022
+- [v1.12.0](https://github.com/PX4/PX4-Autopilot/releases/tag/v1.12.0): July 11th, 2021
+- [v1.11.0](https://github.com/PX4/PX4-Autopilot/releases/tag/v1.11.0): September 7th, 2020
+- [v1.10.0](https://github.com/PX4/PX4-Autopilot/releases/tag/v1.10.0): December 17th, 2019
+- [v1.9.0](https://github.com/PX4/PX4-Autopilot/releases/tag/v1.9.0): May 25, 2019
+- [v1.8.0](https://github.com/PX4/PX4-Autopilot/releases/tag/v1.8.0): June 19th, 2018
+- [v1.7.0](https://github.com/PX4/PX4-Autopilot/releases/tag/v1.7.0): December 15th, 2017
+- [v1.6.2](https://github.com/PX4/PX4-Autopilot/releases/tag/v1.6.2): June 7th, 2017
+- [v1.5.1](https://github.com/PX4/PX4-Autopilot/releases/tag/v1.5.1): December 7th, 2016
+- [v1.4.1](https://github.com/PX4/PX4-Autopilot/releases/tag/v1.4.1): August 6th, 2016
+- [v1.3.1](https://github.com/PX4/PX4-Autopilot/releases/tag/v1.3.1): May 22nd, 2016
+- [v1.2.0](https://github.com/PX4/PX4-Autopilot/releases/tag/v1.2.0): February 21st, 2016
+- [v1.1.0](https://github.com/PX4/PX4-Autopilot/releases/tag/v1.1.0): December 24th, 2015
+- [v1.0.0](https://github.com/PX4/PX4-Autopilot/releases/tag/v1.0.0): August 23rd, 2015
+
+## Release Procedure
+
+### Point release (Major/Minor change)
+
+Point releases are the ones where the major/minor version tag changes. It comes with major new features, hardware supports as well as refactors.
+
+#### Initial branch out deadline
+
+For the initial beta release of a new major/minor change, a "branch out deadline" gets set during the Maintainers Call.
+
+The procedure is:
+
+1. Agreement on branch out deadline: By consensus during Maintainers call
+2. Publication in social media & mailing list about the branch out deadline: By [Ramon](https://github.com/mrpollo)
+3. Project board with the name for the new release gets created: By [Daniel](https://github.com/dagar)
+   1. The board must include a `patch version` field (selectable)
```

Comment:
Shall we keep the version controlling of the Project board as a free form (up to maintainers)?

I put it here although we didn't discuss this because I think we do need a clear beta version controlling method to avoid confusion, and this is one way I want to suggest how to do it. Do you have other suggestions?